info.mumble.Mumble.yml

```diff
@@ -0,0 +1,97 @@
+id: info.mumble.Mumble
+rename-desktop-file: mumble.desktop
+rename-icon: mumble
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: mumble
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-2
+separate-locales: false
+cleanup:
+  - /usr/include
+modules:
+  - name: boost
```

Comment:
Looks like only headers library are required. Take a look at https://github.com/flathub/org.clementine_player.Clementine/blob/492253a161d8bee0d07eb40c4b0b781747b4a75e/org.clementine_player.Clementine.yaml#L32 how to install only these.